portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -5,18 +5,21 @@
 // A comment in the comment list
 public final class Comment {
 
+  private final String author; 
   private final String content;
   private final String currentDate; 
   private final long timestamp;
   
-  public Comment(String content, String currentDate, long timestamp){
+  public Comment(String author, String content, String currentDate, long timestamp){
```

Comment:
I know this isn't this CL, but I suspect I commented  on this previously and it might not have been addressed. Should this be timestampMicros or timestampMillis?

Reply:
Ahh yes, I forgot to change it in the Comment class file to timestampMillis